### ISSUE TITLE
Specialized Accounts: dokku

### ISSUE THREAD

**Issue by josegonzalez** [hachyderm/community#361]
### Which type of specialized account are creating? If more than one apply, please select the primary type.

Open Source Project

### How do you fit in with the community?

Dokku is an open source project dedicated to making deployments easier for folks new to service administration.

### How will you ensure the community is protected?

The account will only directly respond to posts that mention Dokku, and otherwise will be used to post updates about the project and things surrounding the project (Cloud Native Buildpacks, Docker, app deployments, etc.). Examples of our content can be seen on the twitter account: https://twitter.com/dokku

### (Accounts with Automated Posting/Bot Accounts Only)
What method are you using to validate your bots code to ensure that any future updates don't disrupt the community?

Content is not automated at this time - we may mirror content from twitter, but only original posts, and not retweets or replies to others.

**Comment by quintessence** [comment]
👋 Hello! Your project looks wonderful! I was looking at your website / etc. to try and figure out if, as a potentially larger project, dokku would still fall under our OSS policy. Can you send me a quick email to discuss? [admin@hachyderm.io](mailto:admin@hachyderm.io) 

Thank you!

**Comment by josegonzalez** [comment]
I sent an email earlier today, hope to hear from ya'll soon!

**Comment by quintessence** [comment]
Thank you for following up with us!  

You've been added to the approval page. Please follow the instructions on the [Account Verification](https://community.hachyderm.io/docs/account-types/verification/) page to add `https://community.hachyderm.io/approved/` to your profile meta data with the name/label `APPROVED`. Once this is done it should show as green on your profile. If so, please close this ticket. If the verification fails for some reason, please let us know here and leave the ticket open.

**Comment by josegonzalez** [comment]
That works on my end. Should I do something with the site to have that show up green as well like the corp site link does?

**Comment by quintessence** [comment]
No, it should appear green when you add it :) Right now it is showing green for me :)

**Comment by quintessence** [comment]
To clarify: the Community Approved link shows as green and currently your dokku site does not :)
